.github/actions/create-dev-env/action.yml

```diff
@@ -14,3 +14,8 @@ runs:
         pip install --upgrade pip
         pip install --upgrade -r requirements-dev.txt
       shell: bash
+
+    - name: Set Up Docker 🐳
+      uses: docker/setup-docker-action@b60f85385d03ac8acfca6d9996982511d8620a19 # v4.3.0
+      with:
+        set-host: true
```

Comment:
This magic option changes anything - I didn't actually need to change other code around docker client